perf-tests/simulations/src/test/scala/AkkaHttpTapirSimulation.scala

```diff
@@ -0,0 +1,13 @@
+package perfTests
+
+import io.gatling.core.Predef._
+import io.gatling.http.Predef._
+
+class AkkaHttpTapirSimulation extends Simulation {
+  val httpProtocol = http
+    .baseUrl("http://127.0.0.1:8080/akka-http-tapir")
+    .acceptHeader("application/json")
+    .contentTypeHeader("json")
+
+  setUp(perfTests.Common.scn.inject(atOnceUsers(100)).protocols(httpProtocol))
```

Comment:
maybe starting the server (& stopping it later) can be part of the simulations' setup/tear down?

Reply:
also, does this include a warm-up period?